plugins/module_utils/mso.py

```diff
@@ -907,6 +919,15 @@ def make_dhcp_label(self, data):
         ''' Create a DHCP policy from input '''
         if data is None:
             return None
+        if type(data) == list:
+            dhcps = []
+            for dhcp in data:
+                if 'dhcp_option_policy' in dhcp:
+                    dhcp_option_policy = dhcp['dhcp_option_policy']
```

Comment:
can these two lines be put in one like
dhcp['dhcpOptionLabel'] = dhcp['dhcp_option_policy']
?

Reply:
we could

Reply:
dhcp['dhcpOptionLabel'] = dhcp.get('dhcp_option_policy')